sdk/storage/azure-storage-queue/tests/test_queue_async.py

```diff
@@ -220,22 +201,20 @@ def test_list_queues(self):
         self.assertNotEqual('', queues1[0].name)
         # Asserts
         self.assertIsNotNone(queues2)
-        self.assertTrue(len(self.test_queues) - 3 <= len(queues2))
```

Comment:
We should keep some kind of length assertion in the list tests

Reply:
added one.